### PR TITLE
Module implementation ES6

### PR DESCRIPTION
**Pull request by chelobotix**
In this PR, I fulfilled the following:
- [x] Divide the code into modules.
- [x] Refactor methods and functions to arrow functions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,7 +32,7 @@
 
 # 📖 Protoje Concert - Capstone Micorverse Project Module 1<a name="about-project"></a>
```
Comment (omar-muhamad) [inline README.md]: - [x] Please add the correct project name here (Awesome Books) to make your README file look professional.

File: modules/bookCollection.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    window.location.reload();
+  }
+
+  getBooks() {
+    return this.books;
+  }
+
+  setBooks(books) {
+    this.books = books;
+  }
```
Comment (omar-muhamad) [inline modules/bookCollection.js]: - [x] As per this [requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/books/books_with_es6.md#:~:text=Refactor%20your%20methods%20and%20functions%20to%20arrow%20functions.), Please replace these methods with `ES6` arrow function syntax.
[Hint] For example:
```javascript
 addBook = (cTitle, cAuthor) => {
   // functionality here
}
```

File: modules/domManipulation.js [hunk 3]
```diff
@@ -0,0 +1,65 @@
+import BookCollection from './bookCollection.js';
+
+export default function displayBooks() {
+  const newBooksCollection = new BookCollection();
+  const div = document.querySelector('#bookListContainer');
+  const ul = document.querySelector('#ulBookList');
+  const addForm = document.getElementById('addBookForm');
```
Comment (omar-muhamad) [inline modules/domManipulation.js]: - [x] Same here, please use the `ES6` arrow function syntax instead of regular syntax.

**Review by omar-muhamad — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @chelobotix,

Nice job, you are almost there.



## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- You used the correct workflow.  ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [ ] Please check the inline comments below.

## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by chelobotix** [comment]
All observations have been fixed.
Observation 1:
image

Observation 2:
image

Observation 3:
image

**Review by thecodechaser — APPROVED** [review]
## STATUS: APPROVED 🟢

## Hi @chelobotix👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**